package.json

```diff
@@ -4,19 +4,20 @@
 		"test:unit-test": "jest",
 		"test:end-to-end": "node tests/tester.js",
 		"test": "npm run test:end-to-end && npm run test:unit-test",
```

Comment:
```suggestion
		"test": "npm run test:end-to-end && npm run test:unit-test",

```

Reply:
not sure whats up here, should all be tabbed without a starting space.

Reply:
It's the empty line that is removed when you run yarn or npm. That's all. @adamzerella already made the changes for you, you just need to accept them :)